BUILD

```diff
@@ -5,10 +5,7 @@ load("@io_bazel_rules_docker//container:container.bzl", "container_bundle")
 container_bundle(
     name = "all",
     images = {
-        "gcr.io/{PROJECT_ID}/go:latest": "//base:static",
```

Comment:
Oh, actually this is not `static:latest` but `go:latest`. But probably this has to go anyway?

Reply:
We don't actually maintain anything called go anymore, the name change to static happened awhile ago.